.github/workflows/js-css-lint.yml

```
@@ -28,9 +28,19 @@ jobs:
 
       - name: Install dependencies
         run: |
-          # Installs project dependencies using npm in all defined workspaces, if present.
-          # Uses a clean install to ensure a reproducible build environment.
-          npm ci --workspaces --if-present
+          if [ -f package.json ]; then
+            npm install
+            npm install @wordpress/scripts --save-dev
+          fi
+          # Install dependencies in subdirectories
+          for dir in $(find . -type f -name "package.json" -not -path "*/node_modules/*" -exec dirname {} \;); do
+            if [ "$dir" != "." ]; then
+              echo "Installing dependencies in $dir"
+              cd $dir
+              npm install
+              cd -
+            fi
+          done
```

```suggestion
          npm ci
          # Install dependencies in subdirectories
          for dir in $(find . -type f -name "package.json" -not -path "*/node_modules/*" -exec dirname {} \;); do
            if [ "$dir" != "." ]; then
              echo "Installing dependencies in $dir"
              cd $dir
              npm ci
              cd -
            fi
          done
```
How about something like this then?
I think there should be a way for the parent `package.json` to install its own children `package.json`s but I'm not sure and that would require changing whatever repos we have that are structured that way.